### DISCUSSION TITLE
Racer BoM

### DISCUSSION THREAD

**Discussion by Endy01987** [StuckAtPrototype/Racer#46] [Q&A]
Edit:
Was too quick with this post here. Found the answer in the Issues and its supposed to be 5.1kOhm

Hey there,

Looks like a super fun project - thanks alot for making it open source.

I'm going through the BoM's to see where id end up with the costs.
Could someone help me to confirm the Racer BoM ID8, the R22. Is the resistor supposed to be 5.1Ohm or 5.1kOhm? Looks like the supplier reference is not matching the Designation and Note or...  i'm reading it wrong... :)

Cheers!

**Comment by SixAxisSig** [comment]
my board uses 5.1kOhm, based on the issue #39.
